src/examples/poi-crosschecker/poi-crosschecker.ts

```diff
@@ -201,8 +215,7 @@ const run = async () => {
       }
 
       //Q: change cost models dynamically. maybe output divergedDeployment?
-      nPOIs.clear();
-      localnPOIs.clear();
+      db.run("DELETE FROM npois");
```

Comment:
Should we keep some records? If we keep the latest from each sender, then we don't need to keep track of sender nonce anymore?

Reply:
That's exactly right, but the nonce checks happen at the sdk level 😿 so I don't see why we should keep any data after the attestation check, but I'm open to suggestions ofc

Reply:
Actually my thought was to move the db clearing operation to the beginning of the attestation check. That way after a comparison has happened we can make assertions on the db, and clear the db once the next comparison starts

Reply:
But that will be in the PR for the integration test assertions

Reply:
alright cool, we can definitely keep the POI db clean and small, and clean before assert sounds good to me